### PR TITLE
[chore] Add ModuleAttachment, fix accessibility on ListItemHeader

### PR DESCRIPTION
**Pull request by Vangaorth**
## Short description
This PR add the ModuleAttachment component and fixes accessibility on Android for ListItemHeader

## List of changes proposed in this pull request
- Added ModuleAttachment
- ListItemHeader's label ignores accessibility on Android if the parent view is marked as accessibile (this avoid a double reading of such label - iOS is not affected)

### CODE REVIEW

File: src/components/listitems/ListItemHeader.tsx [hunk 1]
```diff
@@ -125,7 +125,10 @@ export const ListItemHeader = ({
       accessible={endElement === undefined ? true : false}
       accessibilityLabel={listItemAccessibilityLabel}
     >
-      <View style={IOListItemStyles.listItemInner}>
+      <View
+        style={IOListItemStyles.listItemInner}
+        importantForAccessibility={endElement ? "auto" : "no-hide-descendants"}
```
Comment (CrisTofani) [inline src/components/listitems/ListItemHeader.tsx:130]: maybe we should handle `elementsHiddenForAccessibility` too in order to have the same behaviour both in iOS and Android, what do you think?

**Review by CrisTofani — CHANGES_REQUESTED** [review]
Apart from required changes, we need to add the new component in the example app and storybook for documentation purpose

File: src/components/listitems/ListItemHeader.tsx (hunk 1 above)
Comment (CrisTofani) [reply src/components/listitems/ListItemHeader.tsx:130]: Perfect! 👌🏻

**Comment by Vangaorth** [comment]
> Apart from required changes, we need to add the new component in the example app and storybook for documentation purpose

Done it!

**Review by CrisTofani — APPROVED** [review]

**Pull request merged**